packages/lit-dev-content/site/tutorials/content/working-with-lists/04.md

```diff
@@ -0,0 +1,79 @@
+Creating a list is also useful for rendering repeated patterns like a game
+board, for instance.
+
+The `range()` directive provides a simple way of creating an iterable of
+incrementing integers which can be used with the `map()` directive to
+programmatically render a list of templates in a convenient manner. Nesting this
+allows creation of multi-dimensional patterns.
+
+The example component already has some styles defined to create an 8 by 8 board
+to be filled with `<div>`s whose background color will be determined by adding
+classes "black" or "white".
+
+Start by importing both directives.
+
+{% switchable-sample %}
+
+```ts
+// my-element.ts
+import {range} from 'lit/directives/range.js';
+import {map} from 'lit/directives/map.js';
+```
+
+```js
+// my-element.js
+import {range} from 'lit/directives/range.js';
+import {map} from 'lit/directives/map.js';
+```
+
+{% endswitchable-sample %}
+
+Use these directives to create squares of black and white colors like a chess
+board.
+
+{% switchable-sample %}
+
+```ts
+// my-element.ts
+  render() {
+    return html`
+      ⋮
+      <div id="board">
+        ${map(range(8), (row) => map(range(8), (col) => html`
+          <div class="${getColor(row, col)}">${getLabel(row, col)}</div>
```

Comment:
I think this is missing `.square` here.

```suggestion
          <div class="square ${getColor(row, col)}">${getLabel(row, col)}</div>
```

Reply:
I actually dropped the square class since all it was doing was adding some padding, and selected for `#board`'s children to add that styling instead. The solution already reflects this but I forgot to change the before codes. This will be fixed in my next push.